hydroshare/urls.py

```diff
@@ -17,69 +17,127 @@
-        name='tracking-report-history'),
-    url(r'^tracking/$', tracking.UseTrackingView.as_view(), name='tracking'),
-    url(r'^tracking/applaunch/', tracking.AppLaunch.as_view(), name='tracking-applaunch'),
-    url(r'^user/$', theme.UserProfileView.as_view()),
-    url(r'^user/(?P<user>.*)/', theme.UserProfileView.as_view()),
-    url(r'^comment/$', theme.comment),
-    url(r'^rating/$', theme.rating),
-    url(r'^profile/$', theme.update_user_profile, name='update_profile'),
-    url(r'^update_password/$', theme.update_user_password, name='update_password'),
-    url(r'^resend_verification_email/(?P<email>.*)/', theme.resend_verification_email,
-        name='resend_verification_email'),
-    url(r'^reset_password_request/$', theme.request_password_reset,
-        name='reset_password_request'),
-    url(r'^new_password_for_reset/(?P<token>[-\w]+)/', theme.UserPasswordResetView.as_view(),
-        name='new_password_for_reset'),
-    url(r'^confirm_reset_password/$', theme.reset_user_password,
-        name='confirm_reset_password'),
-    url(r'^deactivate_account/$', theme.deactivate_user, name='deactivate_account'),
-    url(r'^delete_irods_account/$', theme.delete_irods_account, name='delete_irods_account'),
-    url(r'^create_irods_account/$', theme.create_irods_account, name='create_irods_account'),
-    url(r'^accounts/login/$', theme.login, name='login'),
-    url(r'^landingPage/$', theme.landingPage, name='landing_page'),
-    url(r'^home/$', theme.dashboard, name='dashboard'),
-    url(r'^$', theme.home_router, name='home_router'),
-    url(r'^email_verify/(?P<new_email>.*)/(?P<token>[-\w]+)/(?P<uidb36>[-\w]+)/',
-        theme.email_verify, name='email_verify'),
-    url(r'^email_verify_password_reset/(?P<token>[-\w]+)/(?P<uidb36>[-\w]+)/',
-        theme.email_verify_password_reset, name='email_verify_password_reset'),
-    url(r'^verify/(?P<token>[0-9a-zA-Z:_\-]*)/', hs_core_views.verify),
-    url(r'^django_irods/', include('django_irods.urls')),
-    url(r'^autocomplete/', include('autocomplete_light.urls')),
-    url(r'^search/$', DiscoveryView.as_view(), name='haystack_search'),
-    url(r'^searchjson/$', DiscoveryJsonView.as_view(), name='haystack_json_search'),
-    url(r'^sitemap/$', sitemap, name='sitemap'),
-    url(r'^sitemap', include('hs_sitemap.urls')),
-    url(r'^collaborate/$', hs_core_views.CollaborateView.as_view(), name='collaborate'),
-    url(r'^my-resources/$', hs_core_views.MyResourcesView.as_view(), name='my_resources'),
-    url(r'^my-groups/$', hs_core_views.MyGroupsView.as_view(), name='my_groups'),
-    url(r'^group/(?P<group_id>[0-9]+)', hs_core_views.GroupView.as_view(), name='group'),
-    url(r'^timeseries/sqlite/update/(?P<resource_id>[A-z0-9\-_]+)', hs_ts_views.update_sqlite_file,
-        name='update_sqlite_file'),
-    url(r'^apps/$', hs_core_views.apps.AppsView.as_view(), name="apps")
-)
+if settings.COMMUNITIES_ENABLED:
+    urlpatterns = i18n_patterns(
```

Comment:
This modification in `urls.py` is merely to control the release of the functionality. After everything is live, the `urls.py` file will be collapsed back to a single definition, without the conditional settings.COMMUNITIES_ENABLED, since communities will have been enabled at that point, as the implicit setting.

Reply:
it looks like the urls are mostly the same... could you update it so it reflects which urls are affected?

Reply:
If you're referring to a git diff/readability for the PR, this is actually a full copy. It's not an update of several lines, rather the only way that urls.py would accept the settings.COMMUNITIES_ENABLED parameter as a conditional. I wasn't a fan of this, but given that it's temporary/for the migration and cutover I'm less displeased with is as a solution.